From: David Steele <dsteele@gmail.com>
Date: Fri, 11 Dec 2015 21:52:22 -0500
Subject: Skip browser test if no GUI environment

---
 test/test_browser.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/test/test_browser.py b/test/test_browser.py
index 263bcdc..f3d9065 100644
--- a/test/test_browser.py
+++ b/test/test_browser.py
@@ -1,7 +1,10 @@
 
 import gnomegmail
+import pytest
+import os
 
-
+@pytest.mark.skipif(not os.path.exists("/usr/bin/xdg-settings"),
+                    reason="Not in a true GUI environment")
 def test_browser():
     browser = gnomegmail.browser()
 
